src/v3/src/components/Form/Accordion.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  AccordionDetails,
+  AccordionSummary,
+  AccordionSummaryProps,
+  Box,
+  Typography,
+} from '@mui/material';
+import MuiAccordion from '@mui/material/Accordion';
+import { styled } from '@mui/material/styles';
+import { FunctionComponent, h } from 'preact';
+
+import { AccordionLayout } from '../../types';
+// eslint-disable-next-line import/no-cycle
+import Layout from './Layout';
```

Comment:
Ok for now but let's discuss how to refactor the cycle out. Mind filing a tech-debt Jira?

Reply:
Sure thing! https://oktainc.atlassian.net/browse/OKTA-531082